pyintegration/deephaven2/column.py

```diff
@@ -33,8 +32,8 @@ def __repr__(self):
 class Column:
     """ A Column object represents a column definition in a Deephaven Table. """
     name: str
-    data_type: DType
-    component_type: DType = None
+    data_type: Any
+    component_type: Any = None
```

Comment:
Here is the problem:
The majority of the types defined in dtypes are instances of the DType class, they are not subclasses of DType.  We could make them into classes by defining them as subclasses of DType, but then we need to deal with constructors which don't make sense for the primitive types. Even for DH DateTime etc. - the non-primitive classes, we have a problem with whether to wrap them or not. In the case of receiving a column of DateTime as a Java array, it is best that the data is represented as the jpy type. However a wrapper of that also makes sense because we want to support translation between DH DateTime and the built-in datetime in Python or numpy/pandas specific implementation or some other implementation in other popular Python packages. (I don't know for sure, making that up just for making a point here.)